### DISCUSSION TITLE
[FEATURE REQUEST] intrinsics.type_struct_has_padding :: proc($T: typeid) -> bool where type_is_struct(T) ---

### DISCUSSION THREAD

**Discussion by FourteenBrush** [odin-lang/Odin#3845] [Ideas/Requests]
Would be nice to use this in a compile time assertion, to prevent shooting yourself in the foot when dealing with explicit sizing of structs

**Comment by Kelimion** [comment]
Implemented in 544959326b467b6587a04d6211ea142de932109c
